core/src/test/java/org/frankframework/configuration/digester/TestAnnotationUtils.java

```diff
@@ -219,7 +219,10 @@ protected String buildDefaultBeanName(@Nonnull BeanDefinition definition) {
 		Set<Class<?>> interfazes = new HashSet<>();
 		String[] names = scanner.getRegistry().getBeanDefinitionNames();
 		for (String beanName : names) {
-			if(beanName.contains(this.getClass().getCanonicalName()) || beanName.startsWith("org.frankframework.credentialprovider")) continue; //Ignore this class
+			if(beanName.contains(this.getClass().getCanonicalName())
+					|| beanName.startsWith("org.frankframework.credentialprovider")
+					|| beanName.endsWith(".UnloadableClass")
+			) continue; //Ignore this class
```

Comment:
```suggestion
			) continue; //Ignore this class because credentialProviders don't love class loading???
```
Perhaps a 'why' can be added?